packages/@best/frontend/server/auth.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+*/
+import { Request, Response, NextFunction } from "express";
+import jwt from 'jsonwebtoken';
+
+const TOKEN_SECRET = process.env.TOKEN_SECRET as string;
+const REVOKED_TOKENS = (process.env.REVOKED_TOKENS || "").split("\n");
```

Comment:
> This could also be read from a file. Maybe that would be better. Thoughts?

@dbajric A common convention is to use a `.env` file (see also: [dotenv](https://github.com/motdotla/dotenv#dotenv)).